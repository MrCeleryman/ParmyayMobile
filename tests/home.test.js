import "react-native";
import React from "react";
import renderer from "react-test-renderer";
import Home from "../src/home";

describe("app", () => {
	it("renders correctly", () => {
		const tree = renderer.create(
			<Home />
		);
	});

	it("Snapshot matches", () => {
		expect(renderer.create(
			<Home />
		)).toMatchSnapshot();
	});
});
