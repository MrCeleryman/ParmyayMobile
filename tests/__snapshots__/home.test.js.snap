// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app Snapshot matches 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#FFFCF9",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to Parmyaymobile
  </Text>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#5D34AF",
        "borderRadius": 2,
        "elevation": 5,
        "flexDirection": "row",
        "justifyContent": "center",
        "margin": 5,
        "opacity": 1,
        "padding": 15,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#FFFCF9",
            "fontWeight": "500",
          },
          undefined,
          null,
        ]
      }
    >
      GO TO TEST PAGE
    </Text>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "justifyContent": "center",
        "margin": 5,
        "opacity": 1,
        "padding": 15,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#5D34AF",
            "fontWeight": "500",
          },
          undefined,
          null,
        ]
      }
    >
      FLAT BUTTON
    </Text>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#8FA3AD",
        "borderRadius": 2,
        "elevation": 0,
        "flexDirection": "row",
        "justifyContent": "center",
        "margin": 5,
        "opacity": 1,
        "padding": 15,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#FFFCF9",
            "fontWeight": "500",
          },
          undefined,
          null,
        ]
      }
    >
      RAISED DISABLED
    </Text>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    isTVSelectable={true}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "flexDirection": "row",
        "justifyContent": "center",
        "margin": 5,
        "opacity": 1,
        "padding": 15,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#5D34AF",
            "fontWeight": "500",
          },
          undefined,
          Object {
            "color": "#8FA3AD",
          },
        ]
      }
    >
      FLAT DISABLED
    </Text>
  </View>
</View>
`;
